app/(main)/login/page.test.tsx

```diff
@@ -46,20 +52,37 @@ describe('Login', () => {
     jest
       .spyOn(React, 'useState')
       .mockImplementation(() => [false, setIsLoading]);
+  });
+
+  it('should display GlobalSpinner while authenticating the user', async () => {
+    mockUseAuthContext.isSignedIn = null;
+
+    render(<Login />);
+
+    expect(screen.getByTestId('global-spinner')).toBeInTheDocument();
```

Comment:
Need to test that global-spinner is not in the document as well when isSignedIn is either false or true.